### DISCUSSION TITLE
【寻求帮助】在特定exe中禁用shift切换输入法？

### DISCUSSION THREAD

**Discussion by kola-web** [rime/home#1640] [Q&A]
场景：打游戏的时候经常会用到shift键位，但是会导致频繁切换输入法

**Comment by yrljroli** [comment]
在Win7及之前：

- 按下Ctrl+Space可以切换至输入法「美式键盘」。

在Win11：

- 按下Ctrl+Space可以切换小狼毫至禁用输入法模式，再次按此键可以恢复。
- 按下Win+Space可以切换至其他输入法。在设置中安装输入法「美式键盘」，在想禁用Shift时切换到「美式键盘」即可。

其他系统我未使用过。

**Reply by kola-web** [reply]
多谢你提供的思路
